x-pack/plugins/alerts/server/health/get_health.ts

```diff
@@ -24,6 +24,14 @@ export const getHealth = async (
     },
   };
 
+  if (!internalSavedObjectsRepository) {
+    healthStatuses.decryptionHealth = {
+      status: HealthStatus.Warning,
+      timestamp: new Date().toISOString(),
+    };
+    return healthStatuses;
+  }
```

Comment:
Why have we made `internalSavedObjectsRepository` optional here? Am I missing something?
It doesn't look like we've changed the call to `getHealth` so this looks like it might be leftovers from the original approach?

Reply:
Thank you for catching this leftovers!